### PR TITLE
Reproduce detection of fluorescent lights from legacy capture interface into new capture interface

### PR DESCRIPTION
**Pull request by Georjane**
Fixes #(update with issue number)

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: app/views/capture/index2.html.erb [hunk 1]
```diff
@@ -218,6 +218,7 @@
                   <a rel="tooltip" title="Default mode" class="btn btn-default btn-sm" onClick="$W.toggle_mode()"><i class="fa fa-circle-o"></i></a>
                   <a rel="tooltip" title="RGB mode" class="btn btn-default btn-sm" onClick="$W.toggle_mode()"><i class="fa fa-adjust"></i></a>
                   <a rel="tooltip" title="Combined readout" class="btn btn-default btn-sm" onClick="$W.show_combined()"><i class="fa fa-list-ul"></i></a>
+                  <a rel="tooltip" title="Detected fluorescent light" class="btn btn-default btn-sm" onClick=""><i class="fa fa-lightbulb-o"></i></a>
```
Comment (Georjane) [inline app/views/capture/index2.html.erb:221]: Hi @jywarren @Tlazypanda @TildaDares 
I added this icon to detect fluorescent light on the v2 interface. It is present on the old capture interface but with no functionality. The onClick event listener is empty.

I was thinking that in order to get this to work, we have to add a function to detect fluorescent light into the spectral_workbench.js library before bringing it into the rails apps.

I wish to find out if this function exists already or if not how do I approach writing a function like this?
Was thinking we might have to compare the wavelength of the spectrum to maybe the standard wavelength of LEDs...?

Thank you

Comment (Georjane) [reply app/views/capture/index2.html.erb]: Hi @jywarren, 
Adding `id="cfl-detect"` to the button causes the button to turn red anytime the spectrum detected is a fluorescent light. Do you think this fixes the issue?

**Comment by Georjane** [comment]
@jywarren @Tlazypanda can you please review this PR?
Thank you! 🙏🏽

File: app/views/capture/index2.html.erb (hunk 1 above)
Comment (jywarren) [reply app/views/capture/index2.html.erb]: Yes, that's the correct behavior! Thank you @Georjane - i think this is fixed then!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Woohoo! Well done 🎉

**Comment by jywarren** [comment]
> Hey @Georjane this is in reference to one of the discussions that we had on call where Jeff explained how by checking whether any value matches the peak value array we can determine if it's cfl or not. Here's the code (link is also shared on gitter) for reference - https://github.com/publiclab/spectral-workbench.js/blob/4e3c9f65d51d3dacd4a831ab2aebf39181620db8/examples/capture/api.js#L96-L100

Just circling back on this - Jane and I worked through the code line by line and found that the exact-matching code we had looked at was no longer used to identify CFLs - in fact, it was a simpler approach of looking at "how wiggly" or "steep" the line was. And it was largely working - just needed to be connected back up to the button to notify the user!

**Pull request merged**